.github/workflows/test.yml

```diff
@@ -11,22 +11,6 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    services:
-      mysql:
-        image: mysql/mysql-server:5.6
-        env:
-          MYSQL_ROOT_PASSWORD: rawls-test
-          MYSQL_USER: rawls-test
-          MYSQL_PASSWORD: rawls-test
-          MYSQL_DATABASE: testdb
-        options: >-
-          --health-cmd="mysqladmin ping"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 3306:3306
-
```

Comment:
this method of starting mysql doesn't allow flags such as `--character-set-server=utf8` (or at least I couldn't figure it out) so I abandoned it in favor of reusing the `docker/run-mysql.sh` script.

Reply:
It looks like the port used in `run-mysql.sh` is `3310:3306` which is a little different from the one here `3306:3306`.  Does `run-mysql.sh` need to be updated, or is the discrepancy negligible?